plugin/src/main/java/com/playmonumenta/scriptedquests/quests/components/actions/dialog/DialogAllInOneText.java

```diff
@@ -1,9 +1,11 @@
 package com.playmonumenta.scriptedquests.quests.components.actions.dialog;
 
 import com.google.gson.JsonElement;
+import com.playmonumenta.scriptedquests.api.JsonObjectBuilder;
 import com.playmonumenta.scriptedquests.quests.QuestContext;
 import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.stream.Collectors;
 
 public class DialogAllInOneText implements DialogBase {
 	private ArrayList<DialogAllInOneEntry> mEntries = new ArrayList<DialogAllInOneEntry>();
```

Comment:
```suggestion
	private Collection<DialogAllInOneEntry> mEntries = new ArrayList<DialogAllInOneEntry>();
```

can be downgraded to Collection